### ISSUE TITLE
Etruscan: update the view to zoom to available markers on three of the dropdowns

### ISSUE THREAD

**Issue by jonathanwestin** [gu-gridh/multimodal-map#133]
Use scenario:
1. when the user picks the all-option in the dataset-dropdown, or any of the available options, the view zooms out/in to show all available markers.
2. when the user picks the all option in the site-dropdown, or any of the available options, the view zooms out/in to show all available markers.
3. when the user picks the all option in the necropolis-dropdown, or any of the available options, the view zooms out/in to show all available markers.

In other words, these three dropdowns changes the viewfinder to a position and zoom calculated from a bounding box of the available markers/places. It is exactly how we do it in SHFA with searches, so hopefully that project can function as a blueprint for implementation both in backend and frontend.

**Comment by mtomasini** [comment]
@TristanBridge let me know if you need new views / filters in the backend for this. I think I provided you with what you need but I'm never sure which API you use for what.
